src/steps/access.ts

```diff
@@ -95,38 +119,21 @@ export async function fetchEmployees({
   )) as Entity;
 
   await apiClient.iterateEmployees(async (employee) => {
-    const { hireDate, terminationDate } = await apiClient.getEmployeeDetails(
-      employee.id,
-    );
     const employeeEntity = createIntegrationEntity({
       entityData: {
-        source: {
-          ...employee,
-          hireDate,
-          terminationDate,
-        },
+        source: employee,
         assign: {
           _key: getEmployeeKey(employee.id),
           _type: entities.EMPLOYEE._type,
           _class: entities.EMPLOYEE._class,
-          id: employee.id,
+          ...convertProperties(employee),
```

Comment:
💯  I don't use this as much as I prob could, I forget we have this.

Reply:
I don't love this function, but in the case of this integration for the employees, all the properties on the data are selected by the code - there aren't any properties that we don't want to transfer. 😁